README.md

```
@@ -131,31 +131,88 @@ sqlx migrate run
+./upload "feat: add new feature"
+
+# 查看当前状态
+./upload --status
+
+# 查看帮助
+./upload --help
 ```
 
+**脚本执行流程:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)